<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiSphericalView</class>
 <widget class="QWidget" name="uiSphericalView">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Tool Plots</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="hdrImage"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Image width:</string>
       </property>
       <property name="buddy">
        <cstring>sbWidth</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="sbWidth">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1920</number>
       </property>
       <property name="value">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Image height:</string>
       </property>
       <property name="buddy">
        <cstring>sbHeight</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="sbHeight">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1080</number>
       </property>
       <property name="value">
        <number>128</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="integratorLabel">
       <property name="text">
        <string>Integrator:</string>
       </property>
       <property name="buddy">
        <cstring>integrator</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="integrator">
       <property name="text">
        <string>path</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>SampleCount:</string>
       </property>
       <property name="buddy">
        <cstring>sbSampleCount</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="sbSampleCount">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sbWidth</tabstop>
  <tabstop>sbHeight</tabstop>
  <tabstop>integrator</tabstop>
  <tabstop>sbSampleCount</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
